pkg/kube/roles.go

```diff
@@ -180,3 +187,79 @@ func userRolesFor(userKind string, userName string, envRoles map[string]*v1.Envi
 	}
 	return answer
 }
+
+// CreateClusterRoleBinding creates or extends a cluster role binding in a given namespace for a service account
+func CreateClusterRoleBinding(kubeClient kubernetes.Interface, namespace string, name string,
+	serviceAccountName string, clusterRoleName string) error {
+	_, err := kubeClient.RbacV1().ClusterRoleBindings().Get(name, metav1.GetOptions{})
+	// Create a new role binding if does not exists
+	if err != nil {
+		rb := &rbacv1.ClusterRoleBinding{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       clusterRoleBindingKind,
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
+			},
+			Subjects: []rbacv1.Subject{
+				rbacv1.Subject{
+					Kind:      subjectKind,
+					Name:      serviceAccountName,
+					Namespace: namespace,
+				},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: apiGroup,
+				Kind:     clusterRoleKind,
+				Name:     clusterRoleName,
+			},
+		}
+
+		_, err := kubeClient.RbacV1().ClusterRoleBindings().Create(rb)
+		if err != nil {
+			return errors.Wrap(err, "creating cluster role binding")
+		}
+		return nil
```

Comment:
this line is unnecessary is it will just fall through to the next `return` at the bottom of the function